README.md

```diff
@@ -168,13 +168,15 @@ You can configure a Huly instance to authorize users (sign-in/sign-up) using an
 
 ### On the Huly side
 
-Specify the following environment variables (provided by the IdP) for the account service:
-
+For the account service, set the following environment variables as provided by the IdP:
 * OPENID_CLIENT_ID
 * OPENID_CLIENT_SECRET
 * OPENID_ISSUER
 
-
+To use GitHub OAuth instead, specify:
```

Comment:
1. I wouldn't mix OpenId config with GitHub as you can configure both at the same time and it might be misleading.
2. Please specify the correct redirect URI for GitHub auth as it differs from the OpenId one.

Reply:
Thank you, @lexiv0re. I guess I was thinking that someone would probably not use them both together, but anything is possible!  I'll gladly make your requested changes.

Reply:
@lexiv0re I didn't have to set GITHUB_ISSUER or OPENID_ISSUER in my github.com OAuth setup.  Seems like it was already assumed?

Reply:
For GITHUB all the well-known urls are hardcoded so there's no need in GITHUB_ISSUER. For generic OPENID setting OPENID_ISSUER is mandatory.